logistics_project/apps/malawi/extensions/rapidsms/contact.py

```diff
@@ -44,4 +47,15 @@ def hsa_id(self):
             else:
                 return ""
         else:
-            return ""
\ No newline at end of file
+            return ""
+
+    def translate(self, message):
```

Comment:
oh, i do see [this](https://github.com/dimagi/logistics/blob/malawi-dev/logistics_project/locale/sw/LC_MESSAGES/django.po) though?